tools/config_updater.py

```diff
@@ -553,26 +582,29 @@ def main(unused_argv):
     ShowUser(flags.FLAGS.username, token=token)
 
   elif flags.FLAGS.subparser_name == "update_user":
-    password = None
-    if flags.FLAGS.password:
-      password = getpass.getpass(prompt="Please enter new password for '%s': " %
-                                 flags.FLAGS.username)
-    UpdateUser(flags.FLAGS.username, password, flags.FLAGS.add_labels,
-               flags.FLAGS.delete_labels, token=token)
+    try:
+      UpdateUser(flags.FLAGS.username, flags.FLAGS.password, flags.FLAGS.add_labels,
+                 flags.FLAGS.delete_labels, token=token)
+    except (aff4.InstantiationError):
+      print "User %s does not exist." % flags.FLAGS.username
 
   elif flags.FLAGS.subparser_name == "delete_user":
     DeleteUser(flags.FLAGS.username, token=token)
 
   elif flags.FLAGS.subparser_name == "add_user":
-    if flags.FLAGS.password:
-      password = flags.FLAGS.password
-    else:
-      password = getpass.getpass(
-          prompt="Please enter password for user '%s': " % flags.FLAGS.username)
     labels = []
     if not flags.FLAGS.noadmin:
       labels.append("admin")
-    UpdateUser(flags.FLAGS.username, password, labels, token=token)
+
+    if flags.FLAGS.labels:
+      labels.extend(flags.FLAGS.labels)
+
+    try:
+      if aff4.FACTORY.Open("aff4:/users/%s" % flags.FLAGS.username, "GRRUser",
```

Comment:
I had them inside originally and wound up moving them out, but now I'm not entirely sure why. I think they can probably get moved back in now.